docs/test-scenarios/phase1.md

```diff
@@ -0,0 +1,69 @@
+## Submit and vote on constitution governance action
+
+### Test Steps
+
+1. Produce text file and hash digest of the proposed constitution
```

Comment:
Could this be "proposed SanchoNet constitution" just to make clear that this is for SanchoNet only.